.github/workflows/main.yml

```
@@ -0,0 +1,38 @@
+name: Build Linux
+
+on:
+  push:
+    branches: [ jellyfin ]
```

```suggestion
    branches:
          - jellyfin
```

Your way is correct but imo using sequences is more "Yamly" :)

(Use the tab correctly in your local editor as I think GH messes with it in reviews)

Same in all the other arrays.

---

while I mostly agree, I would argue for single element arrays I actually prefer the Json array syntax (but to each their own :wink: )

---

I don't have a specific preference, but I think we should stay consistent. Since Vue uses the one @ferferga suggestes (https://github.com/jellyfin/jellyfin-vue/blob/master/.github/workflows/docker.yml) we should go with that

---

I used that syntax in Vue basically because imo it's easier to read short lines and keeps all the file within a column that doesn't exceeds too much in size anywhere.

Otherwise it feels descompensated with the job definition :). But this is super extra nitpicky, no need to go for a "standard" anywhere really, so it's up to you.